### PR TITLE
Update how-to-configure-connectors.md

### PR DESCRIPTION
**Pull request by sumeetmittal**
Updating for GCP

### CODE REVIEW

**Comment by sumeetmittal** [comment]
#sign-off

**Comment by Jak-MS** [comment]
@kenwith 
Can you review the proposed changes? 

**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by kenwith** [comment]
#sign-off

**Review by kenwith — APPROVED** [review]

File: docs/global-secure-access/how-to-configure-connectors.md [hunk 1]
```diff
@@ -128,9 +128,9 @@ To use Private Access, install a connector on each Windows server you're using f
 >Starting from the version 1.5.3437.0, having the .NET version 4.7.1 or greater is required for successful installation (upgrade).
 
 > [!Note]
->**Deploy Private Network Connector for Your Azure and AWS Workloads from Marketplace (Preview)**
+>**Deploy Private Network Connector for Your Azure, AWS and GCP Workloads from respective Marketplaces (Preview)**
```
Comment (Jak-MS) [inline docs/global-secure-access/how-to-configure-connectors.md:131]:
```suggestion
>**Deploy Private Network Connector for Your Azure, AWS, and GCP Workloads from respective Marketplaces (Preview)**
```

**Pull request merged**